Update description for db.prepareForReplanning with 0s timeout. (#2983)
neo4j/docs-operations: modules/ROOT/pages/procedures.adoc

.Details
|===
| *Syntax* 3+m| db.prepareForReplanning([ timeOutSeconds ])

> | *Description* 3+a| Triggers an index resample and waits for it to complete, unless the wait time is 0 then it skips waiting, and after that clears query caches. After this procedure has finished queries will be planned using the latest database statistics.

Reviewer: ```suggestion
| *Description* 3+a| Triggers an index resample and waits for its completion. If the wait time is 0, it skips waiting. After that, the procedure clears query caches. Once the process is complete, queries are planned using the latest database statistics.
```
